Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/EncounterTimelineComponent.vue

```diff
@@ -75,7 +75,14 @@ export default class EncounterTimelineComponent extends Vue {
                             Information is from the billing claim and may show a
                             different practitioner or clinic from the one you
                             visited. For more information, visit the
-                            <router-link to="/faq">FAQ</router-link> page.
+                            <a
+                                href="https://www2.gov.bc.ca/gov/content?id=FE8BA7F9F1F0416CB2D24CF71C4BAF80"
+                                target="_blank"
+                                rel="noopener"
+                            >
+                                FAQ
+                            </a>
+                            &nbsp;page.
```

Comment:
If the whitespace inside the `<a>` tag is removed, the `&nbsp;` won't be needed and the link underline will look more natural:

![image](https://user-images.githubusercontent.com/16570293/222608928-2900dd0b-8251-45ed-a3e8-27e088796461.png)

e.g.

```
<a
    href="https://www2.gov.bc.ca/gov/content?id=FE8BA7F9F1F0416CB2D24CF71C4BAF80"
    target="_blank"
    rel="noopener"
    >FAQ</a
>
page.
```

Reply:
I tried to get this to show properly but every time I hit save the automatic editing would mess it up so I resorted to the non-breaking space. Thanks for the secret way to get it to work.